setup_environment.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+###################################################################
+# The purpose of this script is to download, install
+# and configure all files and programs required to
+# take part in developing ActivitySync (Android only).
+###################################################################
+
+CURRENT_USER=$USER
+
+# update package lists from repositories
+sudo apt-get update
+sudo apt-get install git
+
+sudo mkdir /usr/lib/jvm
+cd /tmp/
+
+# download, install and configure java7 jdk
+sudo wget --no-check-certificate --no-cookies --header "Cookie: oraclelicense=accept-securebackup-cookie" http://download.oracle.com/otn-pub/java/jdk/7u79-b15/jdk-7u79-linux-x64.tar.gz
+sudo tar -xvf jdk-7*
+sudo mv ./jdk1.7* /usr/lib/jvm/jdk1.7.0
```

Comment:
Is it really 1.7.0 ? If it is an unknown version from 1.7 family, you could use 1.7.x